docker-images/kafka-based/kafka/scripts/kafka_run.sh

```diff
@@ -77,7 +77,7 @@ if [ "$USE_KRAFT" == "true" ]; then
 
   # Manage the metadata log file changes
   KRAFT_METADATA_LOG_DIR=$(grep "metadata\.log\.dir=" /tmp/strimzi.properties | sed "s/metadata\.log\.dir=*//")
-  CURRENT_KRAFT_METADATA_LOG_DIR=$(ls -d /var/lib/kafka/data-*/kafka-log"$STRIMZI_BROKER_ID"/__cluster_metadata-0 2> /dev/null || true)
+  CURRENT_KRAFT_METADATA_LOG_DIR=$(ls -d /var/lib/kafka/data*/kafka-log"$STRIMZI_BROKER_ID"/__cluster_metadata-0 2> /dev/null || true)
```

Comment:
I thought it's better making consistency with the needed change below. Anyway it's going to match `data-` prefixed folder as well as the `data` folder when persistent-claim is specified.

Reply:
Are you aware of any actual bug? Or are you just increasing the testing scope here? The logic is that unless you use JBOD, there is no moving of the metadata directories. So we do not care about the non-JBOD storage.

Reply:
It's absolutely correct. I didn't explain it well when mentioned the `data` folder for non JBOD. I was making just the two querys consistent but of course I can amend the change on line 80.